apps/aecore/test/aec_test_utils.erl

```diff
@@ -188,7 +188,9 @@ start_chain_db() ->
     ok = mnesia:start(),
     ok = aec_db:initialize_db(ram),
     Tabs = [Tab || {Tab, _} <- aec_db:tables(ram)],
-    ok = mnesia:wait_for_tables(Tabs, 5000).
+    ok = mnesia:wait_for_tables(Tabs, 5000),
```

Comment:
Given my little extension with hooks allowing e.g. `aechannels` to create extra tables (which it currently doesn't), this will not be sufficient in the future.

Reply:
Since we have only one node, I imagine it would be safe to use
`Tabs = mnesia:system_info(tables) -- [schema]`